beetsplug/fetchart.py

```diff
@@ -329,20 +329,36 @@ def get(self, album, plugin, paths):
         if plugin.maxwidth in self.VALID_THUMBNAIL_SIZES:
             size_suffix = "-" + str(plugin.maxwidth)
 
+        url = None
         if 'release' in self.match_by and album.mb_albumid:
-            if size_suffix:
-                release_thumbnail_url = release_url + size_suffix
-                yield self._candidate(url=release_thumbnail_url,
+            url = release_url
+        elif 'releasegroup' in self.match_by and album.mb_releasegroupid:
+            url = release_group_url
+
+        try:
+            response = self.request(url)
+        except requests.RequestException:
+            self._log.debug(u'{0}: error receiving response'.format(self.NAME))
+            return
+
+        try:
+            data = response.json()
+        except ValueError:
+            self._log.debug(u'google: error loading response: {}'
```

Comment:
This log message doesn't seem to describe what's happening here (and references the wrong backend).

Reply:
That, as you can probably tell, was copied from the JSON parsing part from the `GoogleImages` class. If this isn't sufficient, I'm open to suggestions on how to improve it.